cloudflare_worker/worker/src/wasmFunctions.ts

```diff
@@ -62,8 +63,17 @@ interface WasmFunctions {
   validatePayloadHeaders(fields: HeaderFields): void,
 }
 
-export const wasmFunctionsPromise = (async function initWasmFunctions() {
+interface WasmFunctions {
+  init: () => void,
+  WasmWorker: WasmWorker,
+}
+
+export const workerPromise = (async function initWorker() {
   await wasm_bindgen(wasm);
-  wasm_bindgen.init(SXG_CONFIG, CERT_PEM, ISSUER_PEM);
-  return wasm_bindgen as WasmFunctions;
+  const {
+    init,
+    WasmWorker,
+  } = wasm_bindgen as WasmFunctions;
```

Comment:
The initialization of wasm is the same as [an example worker.js](https://github.com/cloudflare/wrangler/blob/7a886341125ded8066047bc2cf658979c8ddddf5/tests/preview.rs#L108-L122) in Wrangler's integration test.
TypeScript compiler does not complain about `wasm_bindgen` and `wasm` because they are declared as `any` type in the very top of `wasmFunctions.ts`. I am adding some more comments to them.

Reply:
Thanks! Those links were exactly what I was looking for (especially ^4).